tensorflow/contrib/tensorrt/convert/convert_graph.cc

```diff
@@ -971,7 +970,7 @@ tensorflow::Status ConvertAfterShapes(ConversionParams& params) {
                                 &graph, alloc.get(), &engine_nodes);
     // If status is ok, we successfully added the node to the graph and can
     // remove segment ops. Otherwise graph is not modified.
-    string msg = StrCat("Engine ", engine.engine_name, " creation for segment ",
+    string msg = StrCat("Adding TensorRT node ", engine.engine_name, " for segment ",
```

Comment:
Since below we'll add the list of nodes and status ("succeeded" or "failed"), how about:

`TensorRT node XXX added for segment X, composed of XX nodes succeeded/failed.`

And log the node names in a separate log line?

Reply:
Done